src/zcash/NoteEncryption.hpp

```diff
@@ -61,13 +61,18 @@ class NoteDecryption {
     typedef boost::array<unsigned char, CLEN> Ciphertext;
     typedef boost::array<unsigned char, MLEN> Plaintext;
 
+    // Unused default constructor to make allocators happy
+    NoteDecryption() { }
```

Comment:
Ah, that's an out-of-date comment, because the default constructor is used to create the argument passed to `GetNoteDecryptor()`.